### PR TITLE
save for later updates

### PR DESCRIPTION
**Pull request by desbo**
Mainly visual changes, including changing the position and adding separators on desktop and reworking the remove buttons in the list view.

@sndrs any thoughts on the sass would be welcome 

screen shot 2015-07-09 at 14 20 24
screen shot 2015-07-09 at 14 20 14
screen shot 2015-07-09 at 14 20 54
screen shot 2015-07-09 at 14 20 41

### CODE REVIEW

File: common/app/views/fragments/items/facia_cards/meta.scala.html [hunk 1]
```diff
@@ -4,17 +4,29 @@
...
     @if(conf.Switches.ABSaveForLaterSwitch.isSwitchedOn) {
-        <button class="js-save-for-later-link fc-save-for-later is-hidden u-faux-block-link__promote" type="button">
-           @fragments.inlineSvg("bookmark", "icon", List("inline-tone-fill"))
-        </button>
+        @if(item.isSavedForLater) {
+            <button class="button button--secondary button--medium saved-content__button js-saved-content__button"
+                    type="button"
+                    data-link-name="Delete"
+                    name="deleteArticle"
+                    type="submit"
```
Comment (sndrs) [inline common/app/views/fragments/items/facia_cards/meta.scala.html]: 2 `type`s

File: identity/app/views/profile/savedForLater.scala.html [hunk 2]
```diff
@@ -19,7 +19,9 @@ <h1 class="fc-container__header__title saved-content_header__title">saved for la
 
                         <div class="saved-content__header__description">you have @(pageData.totalArticlesSaved) saved articles</div>
 
-                        <div class="js-save-for-later__delete-all save-for-later__delete-all-container"></div>
+                        @if(pageData.totalArticlesSaved > 0) {
```
Comment (sndrs) [inline identity/app/views/profile/savedForLater.scala.html]: :nail_care:

File: static/src/stylesheets/module/facia/_item.scss [hunk 3]
```diff
@@ -523,15 +524,21 @@ $block-height: 71px;
 .fc-save-for-later {
     // reset button defaults
     border: 0;
+    border-left: 1px solid;
     background-color: transparent;
     padding: 0;
-
+    outline: none;
```
Comment (sndrs) [inline static/src/stylesheets/module/facia/_item.scss]: this is pretty bad a11y? :eyes:

File: static/src/stylesheets/module/facia/_item.scss [hunk 4]
```diff
@@ -523,15 +524,21 @@ $block-height: 71px;
...
     color: transparent;
-    float: right;
     margin-right: 0 - ($gs-gutter * .25);
     padding-right: ($gs-gutter * .25);
+    padding-left: .75em;
```
Comment (sndrs) [inline static/src/stylesheets/module/facia/_item.scss]: should be a function of `$gs-gutter`

File: static/src/stylesheets/module/facia/_item.scss [hunk 5]
```diff
@@ -539,20 +546,33 @@ $block-height: 71px;
...
     }
 
     .inline-icon {
         float: right;
+        margin-left: $gs-gutter * .25;
+        margin-right: 0;
+
+        @include mq(tablet) {
+            float: left;
+            margin-right: .4em;
```
Comment (sndrs) [inline static/src/stylesheets/module/facia/_item.scss]: `$gs-gutter`

File: static/src/stylesheets/module/identity/_identity.scss [hunk 6]
```diff
@@ -384,6 +384,46 @@
...
+        @include mq(desktop) {
+            position: absolute;
+            top: 0;
+        }
+    }
+
+    .fc-item__content,
+    .fc-item--has-metadata .fc-item__content {
+        @include mq($until: desktop) {
+            padding-bottom: $gs-gutter * 2;
```
Comment (sndrs) [inline static/src/stylesheets/module/identity/_identity.scss]: use `$gs-baseline` for vertical measurement

File: static/src/stylesheets/module/identity/_identity.scss [hunk 7]
```diff
@@ -384,6 +384,46 @@
...
+
+        @include mq(desktop) {
+            right: 0;
+            top: 8px;
+        }
+    }
+
+    .fc-item--has-cutout .saved-content__button {
+        @include mq(tablet) {
+            right: -130px;
```
Comment (sndrs) [inline static/src/stylesheets/module/identity/_identity.scss]: if these are based on hard coded amounts elsewhere, add a comment explaining that and pointing back. probably also a good idea to leave a comment in the original place too

File: static/src/stylesheets/module/identity/_identity.scss [hunk 8]
```diff
@@ -471,7 +515,7 @@
     .inline-icon {
         float: left;
         padding-right: .5em;
-        fill: colour(neutral-2);
+        fill: colour(neutral-2) !important;
```
Comment (sndrs) [inline static/src/stylesheets/module/identity/_identity.scss]: :worried: can this not be avoided?

Comment (desbo) [inline static/src/stylesheets/module/identity/_identity.scss]: not for the time being it seems :(

File: static/src/stylesheets/module/facia/_item.scss [hunk 9]
```diff
@@ -523,15 +524,20 @@ $block-height: 71px;
...
     border: 0;
+    border-left: 1px solid;
     background-color: transparent;
     padding: 0;
-
     color: transparent;
-    float: right;
     margin-right: 0 - ($gs-gutter * .25);
     padding-right: ($gs-gutter * .25);
+    padding-left: $gs-gutter / 4;
```
Comment (sndrs) [inline static/src/stylesheets/module/facia/_item.scss]: id' keep this the same style as the ones around it, don't mind which

**Comment by sndrs** [comment]
:+1:

**Pull request merged**